web/src/components/landing/LandingCanvas.jsx

```diff
@@ -48,18 +48,27 @@ class LandingCanvas extends React.PureComponent {
     this.animate = this.animate.bind(this);
     this.drawHex = this.drawHex.bind(this);
     this.addHexCells = this.addHexCells.bind(this);
+    this.initializeGraphics = this.initializeGraphics.bind(this);
+    this.initWrapper = this.initWrapper.bind(this);
```

Comment:
did you test this/did everything make it into git? No initWrapper function is defined in the component...

Reply:
I changed name of initWrapper to something else before I pushed, forgot to change the name here when I was binding it. Fixed now. Lmk if there are any other problems